core/geometry/src/geometry3d/Point3dVector3d.ts

```diff
@@ -328,24 +379,30 @@ export class XYZ implements XYAndZ {
       return xyz.z;
     return defaultValue;
   }
-
 }
+
 /** 3D point with `x`,`y`,`z` as properties
  * @public
  */
 export class Point3d extends XYZ {
   /** Constructor for Point3d */
-  constructor(x: number = 0, y: number = 0, z: number = 0) { super(x, y, z); }
+  constructor(x: number = 0, y: number = 0, z: number = 0) {
+    super(x, y, z);
+  }
   /**
    * Convert json to Point3d.  Accepted forms are:
    * * `[1,2,3]` --- array of numbers
    * *  array of numbers: [x,y,z]
    * *  object with x,y, and (optional) z as numeric properties {x: xValue, y: yValue, z: zValue}
    * @param json json value.
    */
-  public static fromJSON(json?: XYZProps): Point3d { const val = new Point3d(); val.setFromJSON(json); return val; }
+  public static fromJSON(json?: XYZProps): Point3d {
+    const val = new Point3d(); val.setFromJSON(json); return val;
```

Comment:
If you're making style changes please adhere to our [style guidelines](https://www.itwinjs.org/learning/guidelines/typescript-coding-guidelines/) and put one statement per line, here and elsewhere.

Reply:
sorry! missed that one! there were too many changes! will fix